pandas/tests/indexes/timedeltas/test_timedelta.py

```diff
@@ -509,6 +509,10 @@ def test_fields(self):
         tm.assert_series_equal(s.dt.seconds, Series(
             [10 * 3600 + 11 * 60 + 12, np.nan], index=[0, 1]))
 
+        # preserve name (GH15589)
```

Comment:
> The only problem is that they don't have a common field attribute.

you should simply run it for ``index._datetimelike_ops`` which are defined per-class

but no big deal